yuniql-core/DirectoryService.cs

```diff
@@ -58,7 +59,7 @@ public string[] FilterFiles(string workingPath, string environmentCode, List<str
 
             var hasEnvironmentAwareDirectories = files.Any(f =>
             {
-                var fileParts = Split(new DirectoryInfo(Path.GetDirectoryName(f))).Where(x=> !x.Equals("_transaction", System.StringComparison.InvariantCultureIgnoreCase)).ToList();
+                var fileParts = Split(new DirectoryInfo(Path.GetDirectoryName(f))).Where(x=> !x.Equals(TRANSACTION, System.StringComparison.InvariantCultureIgnoreCase)).ToList();
```

Comment:
@bymse , can you make this semi qualified name such as `RESERVE_DIRECTORY_NAME.TRANSACTION` just to be consistent with how we use thse global constatnt and remove `using static Yuniql.Core.RESERVED_DIRECTORY_NAME`.

I think all else looks good.